cmd/kpod/create.go

```diff
@@ -12,7 +12,6 @@ import (
 	"github.com/pkg/errors"
 	"github.com/projectatomic/libpod/libpod"
 	"github.com/urfave/cli"
```

Comment:
Leave things in this file - they're not used yet, but we're slowly writing code to wire them in

Reply:
done!